### ISSUE TITLE
Install wine

### ISSUE THREAD

**Issue by jlozadad** [Linux-Gamers/fedora_gaming_dev#3]

**Comment by ekultails** [comment]
I have made a role for Wine here: https://github.com/ekultails/ansible-role-wine

We could have it be part of the requirements.yaml

There's a license conflict of Apache v2 vs. GPLv3 so we couldn't bundle it unless I re-license my ansible-role-wine. Defining it in a requirements.yaml should be fine because no files are statically linked (I am not a lawyer, though).

This opens up a larger discussion: should we create different GitHub repositories for each role? And use this project as a collection of Playbook files and dependency requirements?

**Comment by jlozadad** [comment]
We can change the license on this one if you want. I only added that one to avoid the default BSD one.

**Comment by jlozadad** [comment]
Yes I was playing to have this org host a bunch of roles in their own repo
